lib/fastlane/plugin/dynatrace/helper/dynatrace_helper.rb

```diff
@@ -114,6 +114,65 @@ def self.get_dss_client(params)
         return dtxDssClientPath
       end
 
+      def self.symlink_lldb(lldb_path, destination_path, auto_link = true)
```

Comment:
Maybe dd some general log messages so it's traceable e.g. that the symlinking was invoked?